scripts/winesapos-android.sh

```diff
@@ -0,0 +1,14 @@
+echo "This may take a while to install dependencies"
+# This part will install yay
+cd ~/
+sudo pacman -S git
+git clone https://aur.archlinux.org/yay.git
+cd yay
+makepkg -si
+echo "Time To install Android"
+# DONT REMOVE THESE LINES PLEASE THIS WILL BREAK EVERyTHING
+yay -S waydroid
```

Comment:
You can add these packages to our winesapOS repository by adding these lines here:

```
makepkg_fn waydroid install
makepkg_fn waydroid-image-gapps install
makepkg_fn waydroid-image
```

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/repo/winesapos-build-repo.sh#L60